package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nebular",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
let's keep this change separated from this PR

Reply:
Ok, I'll remove.